src/components/inputs/index.tsx

```diff
@@ -195,10 +227,11 @@ export const InputField: React.FC<TextInputProps &
   options,
   touched,
   errors,
+  variant = 'light',
```

Comment:
Maybe avoid default

Reply:
I wanted to avoid breaking the existing implementations :) But we can remove this once we're happy with the new implementation